util/weekly_build/apptests/test_ufswm.sh

```diff
@@ -3,3 +3,19 @@
 # Do some stuff, and exit non-zero if UFSWM cannot be successfully built and
 # tested (ideally, have a discernably different error condition if there are
 # numerical differences)
+
+echo Base directory: ${RUNDIR:?}
+cd ${RUNDIR}
+
+UFSWM_BRANCH=${UFSWM_BRANCH:-develop}
+UFSWM_URL=${UFSWM_URL:-"https://github.com/ufs-community/ufs-weather-model.git"}
```

Comment:
As it stands, will ufs-weather-model be able to pick up on the modules in the installation path we're testing? Or is this going to use the MODULEPATH settings currently in the ufs-weather-model repo?

Reply:
As noted prior:

Changes to `ufs-weather-model` are required to test against a stack built via automation, as `ufs-weather-model/tests/rt.sh` and associated lua files, in the `develop` branch, expect to use a most recently released / published stack:

- `ufs-weather-model/tests/rt.sh`
  - may need to change the value of `dprefix` to a location where the `$USER` running the automated build has `rwx` privileges
- `ufs-weather-module/modulefiles/ufs_<host>.<compiler>.lua` must be changed to reflect:
  - location of automated build stack modules
  - version(s) of modules loaded, if different from past stack releases
    - e.g. newer compiler

Setting `UFSWM_BRANCH` appropriately in `sites/<site>.sh` allows for this; for example, see:

- [ufs_orion.intel.lua](https://github.com/rickgrubin-noaa/ufs-weather-model/blob/spack-stack-automation/modulefiles/ufs_orion.intel.lua)
- [tests/rt.sh](https://github.com/rickgrubin-noaa/ufs-weather-model/blob/spack-stack-automation/tests/rt.sh#L785)